doc/sphinx/conf.py.in

```diff
@@ -86,7 +86,7 @@ language = None
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ["espressomd.gen_code_info*", "espressomd.gen_pxiconfig*"]
+exclude_patterns = ["espressomd.gen_code_info*", "espressomd.gen_pxiconfig*", "espressomd.core.rst"]
```

Comment:
Why `.rst`?

Reply:
because it's the name of the file we want to exclude

Reply:
actually we probably don't need those exclude patterns at all if they are excluded in the ```CMakeLists.txt``` file

Reply:
I can't find it, did you check that in?

Reply:
it gets generated, but probably not if the tool is called correctly